behat/tests/bootstrap/v2/reporting/sections/ActionsSectionTrait.php

```diff
@@ -0,0 +1,143 @@
+    {
+        $this->iViewActionsSection();
+        $this->clickLink('Start actions');
+    }
+
+    /**
+     * @Given I choose no and save on financial decision actions section
+     */
+    public function iChooseNoOnFinancialDecisionActionsSection1()
+    {
+        $this->fillInActionsForm(
+            'no',
+            'action[doYouExpectFinancialDecisions]'
+        );
+    }
+
+    /**
+     * @Given I choose yes and save on financial decision actions section
+     */
+    public function iChooseYesOnFinancialDecisionActionsSection1()
+    {
+        $this->fillInActionsForm(
+            'yes',
+            'action[doYouExpectFinancialDecisions]',
+            'first comment',
+            'action[doYouExpectFinancialDecisionsDetails]'
+        );
+    }
+
+    /**
+     * @Given I choose no and save on concerns actions section
+     */
+    public function iChooseNoOnDoYouHaveConcernsActionsSection()
+    {
+        $this->fillInActionsForm(
+            'no',
+            'action[doYouHaveConcerns]',
+        );
+    }
+
+    /**
+     * @Given I choose yes and save on concerns actions section
+     */
+    public function iChooseYesOnDoYouHaveConcernsActionsSection()
+    {
+        $this->fillInActionsForm(
+            'yes',
+            'action[doYouHaveConcerns]',
+            'second comment',
+            'action[doYouHaveConcernsDetails]'
+        );
+    }
+    public function fillInActionsForm($answer, $actionName, $comment = null, $commentName = null)
+    {
+        $this->selectOption($actionName, $answer);
+        if ($answer === "no") {
+            $this->answeredNo += 1;
+        } elseif ($answer === "yes") {
+            $this->answeredYes += 1;
+        }
+        if ($comment != null) {
+            $this->fillField($commentName, $comment);
+            array_push($this->comments, $comment);
+        }
+        $this->pressButton('Save and continue');
+    }
+
+    /**
+     * @When I should see the expected action report section responses
+     */
+    public function iSeeExpectedActionSectionResponses()
+    {
+        $table = $this->getSession()->getPage()->find('css', 'dl');
+        if (!$table) {
+            $this->throwContextualException('A dl element was not found on the page');
+        }
+        $tableEntry = $table->findAll('css', 'dd');
+        if (!$tableEntry) {
+            $this->throwContextualException('A dd element was not found on the page');
+        }
+        $countNegativeReponse = 0;
+        $countPositiveReponse = 0;
+        foreach ($tableEntry as $entry) {
+            if (trim(strtolower($entry->getHtml())) === "no") {
+                $countNegativeReponse += 1;
+            } elseif (strtolower(trim($entry->getHtml())) === "yes") {
+                $countPositiveReponse += 1;
+            }
+        }
+        assert($countNegativeReponse == $this->answeredNo);
+        assert($countPositiveReponse == $this->answeredYes);
```

Comment:
Could we get a bit of spacing between the blocks here? Makes things a little easier to read. Side note, we should agree on the rules for this and add to pre-commit hooks.